src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -664,18 +654,24 @@ export default class GroupComparisonStore {
     }
 
     readonly mRNAEnrichmentData = makeEnrichmentDataPromise({
-        await:()=>[this.enrichmentsGroup1, this.enrichmentsGroup2,this.mRNAEnrichmentProfile],
+        await:()=>[this.mRNAEnrichmentProfile, this._activeGroupsOverlapRemoved],
```

Comment:
I think it should use activeGroups, rather than activeGroupsOverlapRemoved. I made this change to mutations and cna tabs in an upcoming PR of mine.